tests/integration/pkg/helpers/http_client.go

```diff
@@ -117,6 +117,10 @@ func (h *RetryableHttpClient) withRetries(httpCall func() (*http.Response, error
 	return nil
 }
 
+func (h *RetryableHttpClient) GetHttpClient() *http.Client {
```

Comment:
It's unnecessary for getters to start with Get.

https://go.dev/doc/effective_go#Getters